googleapis/google-cloud-node
# migrate code from googleapis/nodejs-cloud-tpu

- feat: initial launch
- feat: add initial files
- feat: initial samples and tests (#2)
- chore(deps): update dependency sinon to v11 (#4)
- chore: release 1.0.0 (#1)
- fix: make request optional in all cases (#9)
- fix(deps): google-gax v2.17.0 with mTLS (#13)
- chore: release 1.0.1 (#10)
- fix(deps): google-gax v2.17.1 (#15)
- chore: release 1.0.2 (#16)
- fix: Updating WORKSPACE files to use the newest version of the Typescript generator. (#17)
- chore: release 1.0.3 (#18)
- chore(nodejs): update client ref docs link in metadata (#23)
- fix(build): migrate to using main branch (#25)
- chore: release 1.0.4 (#26)
- fix(deps): google-gax v2.24.1 (#27)
- chore: release 1.0.5 (#28)
- feat: turns on self-signed JWT feature flag (#29)
- chore: release 1.1.0 (#30)
- fix: flag module as GA (#32)
- chore: release 1.1.1 (#33)
- chore: relocate owl bot post processor (#37)
- build(node): update deps used during postprocessing (#1243) (#42)
- feat: TPU v2alpha1 public protos (#43)
- chore: release 1.2.0 (#44)
- chore(deps): update dependency @types/node to v16 (#45)
- chore(cloud-rad): delete api-extractor config (#46)
- chore(deps): update dependency sinon to v12 (#47)
- docs(samples): add example tags to generated samples (#48)
- build: add generated samples to .eslintignore (#49)
- docs(node): support "stable"/"preview" release level (#1312) (#52)
- chore: add api_shortname and library_type to repo metadata (#51)
- docs(badges): tweak badge to use new preview/stable language (#1314) (#54)
- test(nodejs): remove 15 add 16 (#1322) (#56)
- build: update copyright year to 2022 (#58)
- chore: update v2.12.0 gapic-generator-typescript (#60)
- chore(deps): update dependency sinon to v13 (#62)
- docs(nodejs): version support policy edits (#1346) (#64)
- docs(samples): include metadata file, add exclusions for samples to handwritten libraries (#65)
- chore: update v2.14.2 gapic-generator-typescript (#69)
- build(node): update client library version in samples metadata (#1356) (#76)
- chore(deps): update dependency @types/mocha to v9 (#77)
- chore(deps): update dependency sinon to v14 (#80)
- build!: update library to use Node 12 (#82)
- chore(main): release 2.0.0 (#83)
- chore(deps): update dependency jsdoc-region-tag to v2 (#86)
- chore(deps): update dependency jsdoc-fresh to v2 (#85)
- feat: support regapic LRO (#87)
- chore(deps): update dependency linkinator to v4 (#90)
- chore(main): release 2.1.0 (#88)
- fix: better support for fallback mode (#91)
- fix: change import long to require (#92)
- fix: do not import the whole google-gax from proto JS (#1553) (#95)
- fix: allow passing gax instance to client constructor (#96)
- fix: preserve default values in x-goog-request-params header (#97)
- feat: Add Secure Boot support to TPU v2alpha1 API (#98)
- test: use fully qualified request type name in tests (#100)
- fix(deps): use google-gax v3.5.2 (#104)
- chore(deps): update dependency @types/node to v18 (#102)
- chore(deps): update dependency jsdoc to v4 (#106)
- fix: regenerated protos JS and TS definitions (#107)
- chore(main): release 2.2.0 (#93)
- build: add release-please config, fix owlbot-config

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

Fixes #<issue_number_goes_here> 🦕
